app/src/pages/settings/AnnotationConfigDialog.tsx

```diff
@@ -0,0 +1,424 @@
+                            <Label>Max</Label>
+                            <Input placeholder="1" />
+                            <FieldError>{error?.message}</FieldError>
+                          </NumberField>
+                        );
+                      }}
+                    />
+                  </>
+                )}
+                {annotationType === "CATEGORICAL" && (
+                  <>
+                    {fields.map((item, index) => (
+                      <Flex
+                        key={item.id}
+                        direction="row"
+                        gap="size-100"
+                        alignItems="start"
+                      >
+                        <Controller
+                          control={control}
+                          name={`values.${index}.label`}
+                          rules={{
+                            required: "Category label is required",
+                          }}
+                          render={({ field, fieldState: { error } }) => (
+                            <TextField
+                              {...field}
+                              aria-label={`Value ${index + 1}`}
+                              isInvalid={!!error}
+                            >
+                              <Input placeholder="Category label" />
+                              <FieldError>{error?.message}</FieldError>
+                            </TextField>
+                          )}
+                        />
+                        <Controller
+                          control={control}
+                          name={`values.${index}.score`}
+                          rules={{
+                            required: "Score is required",
+                          }}
+                          render={({ field, fieldState: { error } }) => (
+                            <NumberField
+                              {...field}
+                              value={
+                                typeof field.value === "number"
+                                  ? field.value
+                                  : undefined
+                              }
+                              aria-label={`Score ${index + 1}`}
+                              isInvalid={!!error}
+                            >
+                              <Flex
+                                direction="row"
+                                gap="size-100"
+                                alignItems="center"
+                              >
+                                <Input placeholder={`${index}`} />
+                              </Flex>
+                              <FieldError>{error?.message}</FieldError>
+                            </NumberField>
+                          )}
+                        />
+                        <Button
+                          type="button"
+                          onPress={() => remove(index)}
+                          variant="quiet"
```

Comment:
for now can we leave this default? I wanna create some guidelines around the usage of quiet. As is it looks probably a bit too out of place.
<img width="737" alt="Screenshot 2025-03-24 at 9 18 52 AM" src="https://github.com/user-attachments/assets/cff3556a-83ec-4826-b53d-99f5bea5eb61" />